test/testdata/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: template-service-broker
+parameters:
+- name: IMAGE
+  value: openshift/origin:latest
+- name: TSB_TEMPLATE_NAMESPACE
+  value: openshift
+- name: NAMESPACE
+  value: openshift-template-service-broker
+- name: KUBE_SYSTEM
+  value: kube-system
+objects:
+
+# to create the tsb server
+- apiVersion: extensions/v1beta1
```

Comment:
> do we have other GA product features built on top of beta apis? are we comfortable w/ the migration implications?

I'm trying to get up with @smarterclayton about why service catalog is a daemonset. Seems wrong and @pmorie isn't sure.  I'd choose a deploymentconfig.